packages/core/src/components/Tabs/Tab/types.ts

```diff
@@ -11,6 +11,8 @@ export type StyledProps = HTMLProps<HTMLButtonElement> &
         tabStyle?: TabStyle;
         fraction?: any;
         tabBackground?: TabBackground;
+        variant?: 'flat' | 'solid';
+        totalTabs?: number;
```

Comment:
We are already calculating total tabs over here -> https://github.com/medly/medly-components/pull/320/files#diff-617a05866df4d11ff896e9c7728057127a2606dea7aa848ae406e6f926664624R23

Reply:
I just added it to the `StyledProps` here because I needed to pass it as a prop to the `Styled.TabWrapper`: https://github.com/medly/medly-components/pull/320/files#diff-34015d090b4f3db2982406e4d108d8d206337fa91c74491b8d3c5f8d69ba2f29R88